playbooks/tags.json

```diff
@@ -1,5 +1 @@
-[
-    "FSRSolutionPack",
-    "malwareAlertUsingSymantecSolutions-1.0.0",
-    "PostCreate"
-]
\ No newline at end of file
+[]
```

Comment:
PostCreate tag is missing here.

Reply:
The playbook start state is changed from PostCreate to Manual that's why removed that tag

Reply:
got it